example_data/sample_bundle.json

```diff
@@ -515,10 +621,10 @@
               {
                 "system": "http://snomed.info/sct",
                 "code": "230387008",
-                "display": "Benign occipital epilepsy of childhood - early onset variant"
+                "display": "Reason for recommending early-onset benign childhood epilepsy"
```

Comment:
I thought that these were all changing to panel app codes across the board?

Reply:
Yes. I had updated this too